jpype/_jexception.py

```diff
@@ -30,9 +30,6 @@ class JException(_jpype._JException, internal=True):
 
     """
 
-    def __str__(self):
-        return str(self.toString())
-
```

Comment:
For reasons that are not quite clear removing this line seems to change the output message slightly even through I don't see a reason.   I removed it as part of a general clean up of unnecessary methods.